### ISSUE TITLE
Feedback - Lesson Contents of the Course WordCamp organizer training missing

### ISSUE THREAD

**Issue by jhimross** [WordPress/Learn#2569]
## Type of feedback
//bug
//content

## Description
Lesson Content of the Course WordCamp organizer training missing

## Step-by-step reproduction instructions (optional)
1. Go to 'https://learn.wordpress.org/course/wordcamp-organizer-training/'
2. Click on 'Take Course'
3. Click the first lesson 'What is a WordCamp'
4. Click the next lesson or any other lessons.

## Screenshots or screen recording (optional)
Watch:- https://share.zight.com/jkupA6zR

**Comment by jonathanbossenger** [comment]
@adamwoodnz, are you able to determine if this course ever had content? It looks like all the lessons are empty?

**Comment by adamwoodnz** [comment]
> [@adamwoodnz](https://github.com/adamwoodnz), are you able to determine if this course ever had content? It looks like all the lessons are empty?

Yeah as far as I can tell from the revisions, it looks like all the lessons have only ever had quizzes, no actual content.

**Comment by adamwoodnz** [comment]
I wonder if this is one of those sets of standalone quizzes that relate to a Tutorial or Online Workshop.

**Comment by jhimross** [comment]
Although the course notes that all questions are based on the WordCamp Organizer Handbook and that the course does not contain original content, this format may still lead to confusion. Learners might feel lost or unsure of how to proceed when they are presented with quiz questions without any accompanying lessons or summaries.

To improve the learner experience, we can consider integrating relevant content from the handbook directly into each lesson. This could include:

- Short summaries or excerpts from the handbook
- Structured lesson content that introduces the topic before the quiz
- Embedded links to the specific handbook section for deeper reading
- Contextual guidance that aligns the quiz with the relevant part of the handbook

This change would help ensure that learners understand where the information is coming from without needing to navigate away or feel like they’ve missed a step. It would also make the course feel more complete, cohesive, and accessible particularly for new contributors or community members exploring WordCamp organizing for the first time.

**Comment by webtechpooja** [comment]
Thanks @jhimross for reporting this, it seems like this is quite similar to our WordCamp Organiser handbook - https://make.wordpress.org/community/handbook/wordcamp-organizer/become-an-organizer/
